gammapy/image/models/core.py

```diff
@@ -110,6 +111,85 @@ def evaluate(lon, lat, lon_0, lat_0, sigma):
         return norm * np.exp(exponent)
 
 
+class SkyElongatedGaussian(SkySpatialModel):
+    r"""Two-dimensional elongated Gaussian model (defined on the plane).
```

Comment:
Not clear to me what "defined on the plane" means. Suggest to keep the first summary line short and remove this here. If you want to clarify, put a sentence further down.